servers/icepvp/metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+    "id": "icepvp",
+    "name": "IcePvP Network",
+    "description": "IcePvP Network is a PotPvP Network focused on OP-Prison and Kitmap",
+    "addresses": [
+        "icepvp.cc"
+    ],
+    "primaryAddress": "icepvp.cc",
+    "minecraftVersions": [
+        "1.7",
+        "1.8"
+    ],
+    "primaryMinecraftVersion": "1.8",
+    "primaryColor": "#00ffff",
+    "secondaryColor": "#ffffff",
+    "primaryRegion": "NA",
+    "regions": [
+        "NA"
+    ],
+    "gameTypes": [
+        "Kitmap",
```

Comment:
This is not one of the following:
PvP, PvE, HCF, Factions, Minigames, Skyblock, Parkour, UHC, Hardcore, Survival, Open World, Prison, Creative, Roleplay, orAdventure.